features/user_can_post.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Create articles
```

Comment:
Are  this `articles` or are these `posts`?

Reply:
done